devices.js

```diff
@@ -4143,7 +4143,15 @@ const devices = [
         ],
         toZigbee: [],
     },
-
+    {
+        zigbeeModel: ['tint Smart Switch'],
+        model: '404021',
+        description: 'Tint Smart Switch',
+        supports: 'on/off',
+        vendor: 'MÃ¼ller Licht',
```

Comment:
Doesn't look like the ü got encoded correctly.

Reply:
That's right. Thanks!